src/item/abstract_item.cpp

```diff
@@ -212,7 +212,7 @@ void AbstractItem::contextMenuEvent(QGraphicsSceneContextMenuEvent* event)
     ungrabMouse(); //Important line! Otherwise the element will be moved after you close the context menu and draw a selection rectangle somewhere.
 
     if (actionSel == actionRemove) {
-        remove();
+        disconnectConnections();
         delete this;
```

Comment:
[`this` cannot be assigned.](http://stackoverflow.com/questions/13476879/how-can-you-assign-a-value-to-the-pointer-this-in-c)